Makefile

```diff
@@ -9,3 +23,13 @@ titond:
 	CGO_ENABLED=0 GOOS=linux go build -o ./build/bin/titond ./cmd/titond/main.go
 	@echo "Done building"
 	@echo "Run \"./build/bin/titond\" to launch titond backend."
+
+image:
+	docker build --build-arg TARGETARCH=amd64 --build-arg TARGETOS=linux -t titond-backend .
+
+image-arm:
```

Comment:
I think we can add image-amd! And we can remove TARGETARCH in `image`. 🙇🏼‍♂️

Reply:
Yes, so image rule is not only for amd64
I updated!